### PR TITLE
feat(reopen investigation): allowing to reopen investigation

### PR DESCRIPTION
**Pull request by MayaSasson**

### CODE REVIEW

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationStatusColumn/InvestigationStatusColumn.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        .catch((error) => {
+            logger.error({
+                service: Service.CLIENT,
+                severity: Severity.HIGH,
+                workflow: 'Ending Investigation',
+                step: `got errors in server result: ${error}`,
+                user: userId,
+                investigation: epidemiologyNumber
+            });
+            Swal.fire({
```
Comment (RoeiRom) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationStatusColumn/InvestigationStatusColumn.tsx:49]: use the useCustomSwal hook

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationStatusColumn/InvestigationStatusColumnStyles.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    columnWrapper: {
+        display: 'flex'
+    },
+    openInvestigatonIcon: {
+        margin: '0 0.5vw'
+    },
+    swalTitle: {
+        fontSize: '1.5vw',
+        fontFamily: 'Assistant',
+    },
```
Comment (RoeiRom) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationStatusColumn/InvestigationStatusColumnStyles.ts:14]: remove the class by using the hook

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 3]
```diff
@@ -249,6 +251,14 @@ const InvestigationTable: React.FC = (): JSX.Element => {
...
             case TableHeadersNames.comment:
                 return <CommentDisplay comment={indexedRow[cellName as keyof typeof TableHeadersNames]} />
+            case TableHeadersNames.investigationStatus:
+                const investigationStatus = indexedRow[cellName as keyof typeof TableHeadersNames];
+                const epidemiologyNumber = indexedRow[TableHeadersNames.epidemiologyNumber];
+                return <InvestigationStatusColumn 
+                investigationStatus={investigationStatus} 
+                epidemiologyNumber={epidemiologyNumber}
+                moveToTheInvestigationForm={moveToTheInvestigationForm}
+                />
```
Comment (RoeiRom) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:261]: indentation

**Review by RoeiRom — CHANGES_REQUESTED** [review]

File: server/src/ClientToDBAPI/InvestigationInfo/mainRoute.ts [hunk 4]
```diff
@@ -129,49 +129,51 @@ investigationInfo.post('/comment', (request: Request, response: Response) => {
...
+                        workflow: 'Ending Investigation',
+                        step: `launching graphql API request to update end time with parameters: ${JSON.stringify({
+                            epidemiologyNumber: response.locals.epidemiologynumber,
+                            investigationEndTime
+                        })}`,
+                        user: response.locals.user.id,
+                        investigation: response.locals.epidemiologynumber
+                    });
+                    graphqlRequest(UPDATE_INVESTIGATION_END_TIME, response.locals, {
+                        epidemiologyNumber: parseInt(response.locals.epidemiologynumber),
```
Comment (ItayBenMoshe) [inline server/src/ClientToDBAPI/InvestigationInfo/mainRoute.ts:174]: maybe use the + syntax instead of parseInt (just to keep consistency)

**Review by ItayBenMoshe — DISMISSED** [review]

File: server/src/ClientToDBAPI/InvestigationInfo/mainRoute.ts (hunk 4 above)
Comment (MayaSasson) [reply server/src/ClientToDBAPI/InvestigationInfo/mainRoute.ts]: I accidently didnt change it to use the variable props then I did it now

**Review by RoeiRom — APPROVED** [review]

**Review by ItayBenMoshe — APPROVED** [review]

**Pull request merged**